cli.js

```diff
@@ -1,13 +1,18 @@
 #!/usr/bin/env node
-'use strict';
-const arrify = require('arrify');
-const meow = require('meow');
-const getStdin = require('get-stdin');
-const imagemin = require('imagemin');
-const ora = require('ora');
-const plur = require('plur');
-const stripIndent = require('strip-indent');
-const pairs = require('lodash.pairs');
+
+import {Buffer} from 'node:buffer';
+import {createRequire} from 'node:module';
+import process from 'node:process';
+import arrify from 'arrify';
+import meow from 'meow';
+import getStdin from 'get-stdin';
+import imagemin from 'imagemin';
+import ora from 'ora';
+import plur from 'plur';
+import stripIndent from 'strip-indent';
+import pairs from 'lodash.pairs';
+
+const require = createRequire(import.meta.url);
```

Comment:
`require` should be replaced by `await import(…)`.

Reply:
`require('name')` searches for the module up the directory tree by diving into every`node_modules/<name>`. Аnd then it will also check the `name` existence as global package. Dynamic `import` checks just a single path.

Reply:
> Dynamic import checks just a single path.

Are you sure about that? According to https://nodejs.org/api/esm.html#esm_resolver_algorithm_specification, static `import` statement searches up, so I would have assumed dynamic import does the same.

Reply:
Wow! My experience has been based on early polyfills. It's great that it just works now.